enhancements/console/customize-catalogs.md

```diff
@@ -0,0 +1,199 @@
+---
+title: customize-catalogs
+authors:
+  - "@lokanandaprabhu"
+reviewers:
+  - "@jerolimov"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2022-07-27
+last-updated: 2022-09-01
+tracking-link: 
+  - https://issues.redhat.com/browse/ODC-6695
+status: implementable
+---
+
+# Customize Catalogs
+
+## Summary
+
+The OpenShift Console provides a "Developer Catalog" which enables 
+the user to create apps and services based on Operator backed services, helm charts, templates, etc.
+
+Cluster administrators can provide additional catalog types (providers) 
+via console plugins, for example by installing an operator. 
+These catalog types (providers) extend the catalog with additional 
+items and are also available as "sub-catalogs".
+
+Some customers want to hide specific types from 
+the catalog or the sub-catalog to streamline their 
+development process without uninstalling a complete operator or disabling the complete console plugin.
+
+In the developer catalog the sub-catalogs are shown as types. 
+The developer catalog shows items from all sub-catalogs.
+When user selects a sub-catalog, they can see a list of 
+available items from this sub-catalog to install.
+
+This enhancement allows cluster admins to selectively 
+disable a sub-catalog and all links to the specific sub-catalog. 
+The developer catalog will also hide any of the items from this sub-catalog.
+
+![Developer catalog and types list](customize-catalogs.png)
+ 
+## Motivation
+
+The developer catalog can be extended by plugins and 
+also dynamically by installing a console plugin or an operator.
+
+Some customers want to align their development process and 
+have teams that support other development teams with their work.
+For this cluster administrators want to hide developer sub-catalog (types) 
+without disabling a full console plugin, for example by uninstalling an operator.
+By hiding for example "Service Backed operators", "Helm", etc. 
+they can still install them via the CLI, see their resources in the topology, 
+but don't show them in the add from developer catalog flow.
+
+### User Stories
+
+#### Story 1
+
+As a cluster admin, I want to hide Developer catalog for all users.
```

Comment:
So that... This is missing the why of the user story, so that the catalog is less cluttered? So that admins can curate the available operators for their users?

Reply:
Updated with description.